drivers/at30tse75x/Kconfig

```diff
@@ -10,6 +10,6 @@ config MODULE_AT30TSE75X
     depends on HAS_PERIPH_I2C
     depends on TEST_KCONFIG
     select MODULE_PERIPH_I2C
-    select MODULE_XTIMER
+    select MODULE_ZTIMER_USEC
```

Comment:
```suggestion
    select MODULE_ZTIMER
    select MODULE_ZTIMER_USEC
```

Reply:
Please squash right away when this is addressed!

Reply:
And re-trigger ci!

Reply:
doesn't select MODULE_ZTIMER_USEC depend on MODULE_ZTIMER